
2021-06-08 22:39:00,308 INFO: errors [in d:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:509]
2021-06-08 22:39:36,300 ERROR: Exception on /shows/create [GET] [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "d:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py", line 477, in create_shows
    form = ShowForm()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 157, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 87, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 274, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 131, in process
    field.process(formdata)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\csrf\core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 139, in generate_csrf_token
    return generate_csrf(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 33, in generate_csrf
    secret_key = _get_config(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 128, in _get_config
    raise RuntimeError(message)
RuntimeError: A secret key is required to use CSRF.
2021-06-08 22:39:39,446 ERROR: Exception on /artists/create [GET] [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "d:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py", line 412, in create_artist_form
    form = ArtistForm()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 157, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 87, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 274, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 131, in process
    field.process(formdata)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\csrf\core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 139, in generate_csrf_token
    return generate_csrf(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 33, in generate_csrf
    secret_key = _get_config(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 128, in _get_config
    raise RuntimeError(message)
RuntimeError: A secret key is required to use CSRF.
2021-06-08 22:39:42,665 ERROR: Exception on /venues/create [GET] [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "d:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py", line 219, in create_venue_form
    form = VenueForm()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 157, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 87, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 274, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 131, in process
    field.process(formdata)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\csrf\core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 139, in generate_csrf_token
    return generate_csrf(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 33, in generate_csrf
    secret_key = _get_config(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 128, in _get_config
    raise RuntimeError(message)
RuntimeError: A secret key is required to use CSRF.
2021-06-08 22:47:22,609 INFO: errors [in d:/Fullstack/FSND/projects/01_fyyur/starter_code/app.py:512]
2021-06-08 22:47:32,330 ERROR: Exception on /artists/create [GET] [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "d:/Fullstack/FSND/projects/01_fyyur/starter_code/app.py", line 415, in create_artist_form
    form = ArtistForm()
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 157, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\form.py", line 87, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 274, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\form.py", line 131, in process
    field.process(formdata)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\wtforms\csrf\core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 139, in generate_csrf_token
    return generate_csrf(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 33, in generate_csrf
    secret_key = _get_config(
  File "D:\Fullstack\FSND\projects\01_fyyur\starter_code\env\lib\site-packages\flask_wtf\csrf.py", line 128, in _get_config
    raise RuntimeError(message)
RuntimeError: A secret key is required to use CSRF.
2021-06-08 22:48:11,152 INFO: errors [in d:/Fullstack/FSND/projects/01_fyyur/starter_code/app.py:512]
2021-06-08 22:51:25,080 INFO: errors [in d:/Fullstack/FSND/projects/01_fyyur/starter_code/app.py:512]
2021-06-08 22:52:36,917 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:512]
2021-06-08 22:58:38,347 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:512]
2021-06-08 22:59:05,773 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:512]
2021-06-08 22:59:25,676 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:521]
2021-06-08 23:03:03,603 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:521]
2021-06-08 23:03:20,742 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:521]
2021-06-08 23:03:55,730 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:521]
2021-06-08 23:11:02,568 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:521]
2021-06-08 23:13:43,351 INFO: errors [in D:\Fullstack\FSND\projects\01_fyyur\starter_code\app.py:530]
